.changeset/honest-glasses-drum.md

```diff
@@ -0,0 +1,6 @@
+---
+'@openfn/language-salesforce': patch
+---
+
+- add `query()` limits
```

Comment:
We need to do a version bump in a minute. Before we do, can I picky and say that we haven't added any limits - we've just documented existing ones.

Maybe we can simply say:  

> Add documentation about Salesforce API limits to query and bulkQuery